velox/connectors/hive/HivePartitionFunction.h

```diff
@@ -28,6 +28,16 @@ class HivePartitionFunction : public core::PartitionFunction {
       std::vector<column_index_t> keyChannels,
       const std::vector<VectorPtr>& constValues = {});
 
+  HivePartitionFunction(
```

Comment:
When is this used? It looks like the "spec" object always passes bucketToPartition map,  no?

Reply:
Explained offline. Thanks!